knpu/ep1-intro/routing-wildcards.md

```diff
@@ -17,6 +17,8 @@ The important thing is that the routing wildcard matches the variable name.
 
 To test this is, change the message in the response to `'The genus: '.$genusName`:
 
+**NOTE:** Returning a raw response should only be done in development as it will introduce a security issue called XSS. You can [read more about XSS here.](https://www.owasp.org/index.php/Cross-site_Scripting_%28XSS%29)
```

Comment:
Better to use our internal **TIP** block:
```
***TIP
// your tip here...
***
```

You can get example from [downloading-the-project](https://github.com/knpuniversity/symfony/blob/master/knpu/ep1-intro/start-project.md#downloading-the-project)  chapter.